internal/datasystem/store.go

```diff
@@ -0,0 +1,252 @@
+package datasystem
+
+import (
+	"sync"
+
+	"github.com/launchdarkly/go-server-sdk/v7/internal/memorystorev2"
+
+	"github.com/launchdarkly/go-server-sdk/v7/internal/fdv2proto"
+
+	"github.com/launchdarkly/go-sdk-common/v3/ldlog"
+	"github.com/launchdarkly/go-server-sdk/v7/interfaces"
+	"github.com/launchdarkly/go-server-sdk/v7/subsystems"
+	"github.com/launchdarkly/go-server-sdk/v7/subsystems/ldstoretypes"
+)
+
+// Store is a dual-mode persistent/in-memory store that data queries from the evaluation
+// algorithm.
```

Comment:
`that data queries from` doesn't make sense to me. Can we re-word that?

Reply:
Ah, was missing a word. 👍